api/sixtyfps-rs/migration.md

```diff
@@ -59,3 +59,4 @@ fn model_tracker(&self) -> &dyn ModelTracker {
 ### Interpreter
 
 `Value::Array` was removed and `Value::Model` needs to be used instead.
+`CallCallbackError` was renamed to `InvokeCallbackError`.
```

Comment:
Ah well, technically the docs for the interpreter should go in the interpreter's doc.

Reply:
I don't quite understand. `InvokeCallbackError` is an Rust interpreter specific API that is here mentioned in the interpreter specific section.

Reply:
Ohh, I see, this is the migration doc for the Rust API create, not for the interpreter. Hmm.

Reply:
Let's tackle it when we integrate `migration.md` into the actual docs (not linked yet) -> #888